pkgs/tools/misc/iay/default.nix

```diff
@@ -0,0 +1,24 @@
+{ lib
+, rustPlatform
+, fetchCrate
+, stdenv
+}:
+
+rustPlatform.buildRustPackage rec {
+  pname = "iay";
+  version = "0.4.0";
+
+  src = fetchCrate {
+    inherit pname version;
+    sha256 = "sha256-xp+9WmzvuYf7vfNGTc/ZQ/fF8UsprS3S1SBHPYX/5fg=";
+  };
+
+  cargoSha256 = "sha256-SMqiwM6LrXXjV4Mb2BY9WbeKKPkxiYxPyZ4aepVIAqU=";
+
+  meta = with lib; {
+    description = "Minimalistic shell prompt";
+    homepage = "https://github.com/aaqaishtyaq/iay";
+    license = licenses.mit;
+    maintainers = with maintainers; [ omasanori ];
+  };
```

Comment:
I think you need to set platforms in here (https://nixos.org/manual/nixpkgs/stable/#var-meta-platforms).

Reply:
Darwin builds with
```nix
buildInputs = lib.optionals stdenv.isDarwin [
  darwin.apple_sdk.frameworks.Security
];
```

Reply:
Thank you for reviewing, @deejayem and @raphaelr ! I will address them.

Reply:
Done!